webapp/views.py

```diff
@@ -816,7 +816,14 @@ def make_renewal(advantage, contract_info):
         key=lambda renewal: dateutil.parser.parse(renewal["start"]),
     )
 
-    renewal = sorted_renewals[0]
+    open_renewals = [
+        renewal for renewal in sorted_renewals if renewal["status"] != "closed"
```

Comment:
We could target just pending renewals rather than "not closed", and also do that while sorting, in a single pass:
```python
sorted_renewals = sorted(
    r for r in renewals if r["status"] == "pending", 
    key=lambda r: dateutil.parser.parse(r["start"]),
)
```
What do you think?

Reply:
We have a case in there where a renewal that is "done" can be returned, so that we can show that contract as having been recently renewed (it will have a "Renewed" label on it), and this would stop that from happening - that said, that's another case where if there are multiple renewals to be renewed, the user could renew the first, it would be marked as "done", and then they wouldn't be able to see their second renewal...

Reply:
Agreed that this is out of this PR's scope, and will be addressed in https://github.com/canonical-web-and-design/ubuntu.com/issues/8755